launch/hooks.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+
+class PostInferenceHooks(str, Enum):
+    INSIGHT = "INSIGHT"
```

Comment:
do python standards prefer upper case? if not we can just make this lowercase just like the equals check

Reply:
sure we can do lower-case